core/src/main/scala/kafka/server/Kip500Broker.scala

```diff
@@ -17,16 +17,77 @@
 
 package kafka.server
 
+import java.util.concurrent.locks.ReentrantLock
+
 import kafka.metrics.KafkaMetricsReporter
 import org.apache.kafka.common.utils.Time
+import org.apache.kafka.metadata
```

Comment:
getting this error with Scala 2.12

```
[Error] /Users/rdagostino/git/confluentinc/kafka2/core/src/main/scala/kafka/server/Kip500Broker.scala:90: type BrokerState is not a member of package kafka.server.metadata
[Warn] /Users/rdagostino/git/confluentinc/kafka2/core/src/main/scala/kafka/server/Kip500Broker.scala:24: imported `metadata' is permanently hidden by definition of package metadata in package server
```

```suggestion
import org.apache.kafka.metadata.BrokerState
```
 And then referring to just `BrokerState` below instead of `metadata.BrokerState` seems to fix it.